FirebaseRemoteConfig/Sources/FIRRemoteConfig.m

```diff
@@ -619,4 +623,54 @@ - (FIRConfigUpdateListenerRegistration *)addOnConfigUpdateListener:
   return [self->_configRealtime addConfigUpdateListener:listener];
 }
 
+#pragma mark - Rollout
+
+- (void)addRemoteConfigInteropSubscriber:(id<FIRRolloutsStateSubscriber>)subscriber {
+  [[NSNotificationCenter defaultCenter]
+      addObserverForName:RolloutsStateDidChangeNotificationName
+                  object:self
+                   queue:nil
```

Comment:
Here is documentation for this method: https://developer.apple.com/documentation/foundation/nsnotificationcenter/1411723-addobserverforname

Be careful it is state: When we declare queue as nil, the block runs synchronously on the posting thread. So be careful where you did the post notification. If you did sync on a serial queue once and inside doing sync within the block on the same queue, it will cause deadlock: https://stackoverflow.com/questions/23939730/dispatch-sync-inside-dispatch-sync-causes-deadlock

Reply:
Thank you for pointing this! I changed to get the value from the activeConfig directly instead of call that dispatch method.